src/loader.rs

```diff
@@ -43,16 +31,11 @@ impl AssetLoader for SvgAssetLoader {
             debug!("Parsing SVG: {} ... Done", load_context.path().display());
 
             debug!("Tessellating SVG: {} ...", load_context.path().display());
-            let buffer = tessellation::generate_buffer(
-                &svg,
-                &mut FillTessellator::new(),
-                &mut StrokeTessellator::new(),
-            );
+            let mesh = svg.tessellate();
```

Comment:
Optionally we might want to tessellate in the `from_bytes` function?

Reply:
I'd like to keep it separate. If you tessellate in `from_bytes` you would need to return the buffer/mesh or pass the `LoadContext` as a parameter. Also, tessellation could get computationally expensive and maybe we want to remove it from the loader in the future, depending on how the reworked asset system will work. So, let's keep it the way you did it here, that gives us a little bit more flexibility, if we need it.